torch/_functorch/aot_autograd.py

```diff
@@ -558,7 +562,7 @@ def convert(idx, x):
                     ctx = _detect_attribute_assignment(mod)
                 else:
                     ctx = nullcontext()
-                with ctx:
+                with ctx, maybe_suppress():
```

Comment:
I think the scope of the `maybe_suppress` is too aggressive here. I only want to suppress when I'm creating FunctionalTensors; everything else I still want normal guards to be reported